### ISSUE TITLE
Compilation hangs since 0.56.0

### ISSUE THREAD

**Issue by gaearon** [rollup/rollup#2127]
Repro (requires Yarn >= 1.3.2):

```
git clone https://github.com/gaearon/react.git # my fork
cd react
git checkout rollup-bug-056
yarn

yarn build dom-client --type=UMD_DEV
```

It hangs.

If you downgrade Rollup to 0.55.5 (`yarn add rollup -W --dev 0.55.5`), it succeeds.

Reproes with 0.58.0 too.

Config is [here](https://github.com/gaearon/react/blob/d89654ef1fb3280502923255ea8178ce5ca248f1/scripts/rollup/build.js#L356-L383).

**Comment by lukastaegert** [comment]
Thanks! Might take a few days until I can take a look, though.

**Comment by lukastaegert** [comment]
Initial findings: The hang happens during rendering. More precisely, it happens while rendering `packages/react-dom/src/client/ReactDOMFiberComponent.js` and seems to be related to the new comment rendering logic introduced in 0.56.0. If I remove all the comments in this file, the build completes. Will keep on investigating...

**Comment by lukastaegert** [comment]
BTW you really seem to like your comments...

**Comment by lukastaegert** [comment]
Ok, if you remove the comment directly before the first import, everything works. Will keep on investigating tomorrow. Thanks for bringing this up!

**Comment by gaearon** [comment]
Thanks!

**Comment by lukastaegert** [comment]
Found the issue, it was the "/*" inside the comment which was misinterpreted as a block comment start. Fix here: #2129
